napari/_qt/containers/_tests/test_qt_layer_list.py

```diff
@@ -191,6 +193,60 @@ def make_qt_layer_list_with_delegate(qtbot):
     return image1, image2, image3, layers, view, delegate
 
 
+@skip_local_focus
+def test_drag_and_drop_layers(qtbot):
+    """
+    Test drag and drop actions with pyautogui to change layer list order.
+
+    Notes:
+        * For this test to pass locally, you need to give the Terminal/iTerm
```

Comment:
```suggestion
        * For this test to pass locally on macOS, you need to give the Terminal/iTerm
```
Not sure what the situation would be for other platforms? Does it JustWork?

Reply:
Actually that's a good question :+1: as far as I checked on Windows seems like things just work, on Linux is the same although when running the tests over the CI a need for an `.Xauthority` file raised (which is being taken care with a validation at the `conftest.py` file thanks to https://github.com/napari/napari/pull/6719).

I will apply your suggestion but let me know if putting further info in the comment could be worthy!